src/main/java/racing/domain/Car.java

```diff
@@ -53,28 +46,33 @@ public String getName() {
         return this.name.getName();
     }
 
+    public String getSkidMark() {
+        String skidMark = getName() + " : ";
+        for (int i = 0; i < this.getPosition(); i++) {
+            skidMark += this.getTireMark();
+        }
+        return skidMark;
+    }
+
+    public boolean isWinner(int winnerPosition) {
+        return this.position.equals(new CarPosition(winnerPosition));
+    }
```

Comment:
매개변수로 그냥 CarPosition을 받도록 한다면 객체의 내부값을 꺼내 연산을 할 필요가 없겠죠. 
그리고 이전에 동일한 피드백을 드렸는데, isWinner라는 메서드가 정말 우승자인지를 알려 줄 수는 없는 메서드입니다. 
해당 자동차의 위치 값이 1이고 우승자의 위치값이 10인데, isWinner(1)을 전달하면 값은 true로 우승자라고 반환을 합니다. 그렇기에 해당 메서드 명은 isWinner가 아니라 isEqualPosition이 더 적절하겠죠. 

```suggestion
    public boolean isEqualPosition(CarPosition carPosition) {
        return this.position.equals(carPosition);
    }
```